src/images/body.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 166 380" style="enable-background:new 0 0 166 380;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#C0D5D0;}
+</style>
+<path class="st0" d="M148,182l-1.6,14.8c-0.2,1.5-2,2.3-3.2,1.3l-3.7-3.1c-0.4-0.3-0.6-0.7-0.7-1.2l-1.4-10.3l-3.5,6l-3,1.5l2-5
+	l-0.5-8l1.3-3c1.7-3.9,1.4-8.5-0.7-12.2l-9.9-17.2c-1.6-2.8-2.5-5.8-2.7-9l-0.7-12.2l-7.8-17.8l-3.4,14.4
+	c-1.8,7.8-1.6,15.9,0.8,23.5l4.7,15.4c2.7,8.9,3.6,18.2,2.4,27.4l-5.3,42.5l-3,16.5l5.7,18.1c1.2,3.9,1.5,8,1,12l-2.4,16.6l0,0.1
+	l-1.8,32l6.4,16.2c1,2.6-0.3,5.6-3,6.5l-9.5,3.2c-3.4,1.1-6.8-1.5-6.5-5.1l1.2-15.3l3-9.5l-8.6-41.5v0l0,0
+	c-1.9-6.4-2.2-13.1-0.9-19.6l1.5-7.6l-10.5-76l0,0c-0.8,0.6-1.3,1.6-1.5,2.6L72,253.5l1.5,7.6c1.3,6.5,1,13.3-0.9,19.6l0,0v0
+	L64,322.3l3,9.5l1.2,15.3c0.3,3.5-3.2,6.2-6.5,5.1l-9.5-3.2c-2.7-0.9-4.1-3.9-3-6.5l6.4-16.2l-1.8-32l0-0.1l-2.4-16.6
+	c-0.6-4-0.2-8.1,1-12l5.7-18.1L55,231l-5.3-42.5c-1.1-9.2-0.3-18.6,2.4-27.4l4.7-15.4c2.4-7.6,2.6-15.8,0.8-23.5l-3.4-14.4
+	l-7.8,17.8l-0.7,12.2c-0.2,3.2-1.1,6.3-2.7,9l-9.9,17.2c-2.1,3.7-2.4,8.2-0.7,12.2l1.3,3l-0.5,8l2,5l-3-1.5l-3.5-6l-1.4,10.3
+	c-0.1,0.5-0.3,0.9-0.7,1.2l-3.7,3.1c-1.2,1-3,0.3-3.2-1.3L18.3,183l4.6-6.9c1.8-2.7,2.9-5.9,3.2-9.1l2.6-27.3
+	c0.2-2.1,0.8-4.2,1.7-6.1l6.1-12.6l5.9-39.3c0.4-2.7,1.8-5.1,3.8-6.9l5.9-5c2.2-1.8,4.9-2.7,7.8-2.6h0c3,0.2,5.8-0.9,8-2.8l4.9-4.4
+	c1-0.9,1.5-2.3,1.2-3.7l-0.4-1.9c-0.2-0.8-0.6-1.5-1.2-2c-4.8-4-7.9-10.5-7.9-17.8c0-12.2,8.5-22,19-22s19,9.8,19,22
+	c0,7.8-3.5,14.7-8.8,18.6c-0.7,0.5-1.2,1.3-1.4,2.2v0c-0.3,1.4,0.2,2.8,1.2,3.7l4.9,4.4c2.2,2,5.1,3,8,2.8h0
+	c2.8-0.2,5.6,0.8,7.8,2.6l5.9,5c2.1,1.7,3.4,4.2,3.8,6.9l5.9,39.3l6.1,12.6c0.9,1.9,1.5,3.9,1.7,6.1l2.6,27.3
+	c0.3,3.3,1.4,6.4,3.2,9.1L148,182z"/>
+</svg>
```

Comment:
all your svg could be optimized. i like to use [this tool](https://jakearchibald.github.io/svgomg/) because it lets you paste in the markup and see the actual svg as you optimize it while seeing the compression offered

Reply:
Also if you like a command line tool for this sort of thing you can use: [svgo](https://github.com/svg/svgo)

Reply:
yeah svgo is clutch for batch optimizing
` svgo -f icon/folder/path`